packages/kolibri-components/src/content/KContentPlugin.js

```diff
@@ -0,0 +1,44 @@
+import setupPluginMediator from '../utils/pluginMediator';
+import KContentRenderer from './KContentRenderer';
+import { RENDERER_SUFFIX } from './constants';
+
+export default {
+  install(
+    Vue,
+    {
+      languageDirection,
+      ContentRendererLoadingComponent = {
+        render(h) {
+          return h('p', '-----');
+        },
+      },
+      ContentRendererErrorComponent = {
+        render(h) {
+          return h('p', 'xxxxx');
+        },
+      },
+      facade,
```

Comment:
This could probably just be called `coreAppGlobal` or something?

If I'm understanding correctly, the fact that the API is implemented using a 'facade' pattern is an internal implementation detail of the core app which should not be exposed to the API users.

Reply:
Fair enough - also, this is only provided for use inside Kolibri, as the `undefined` behaviour below for the `facade` argument is required when using this outside of Kolibri.